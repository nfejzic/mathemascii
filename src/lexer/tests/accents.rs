use crate::lexer::tests::Snapshot;

use super::super::*;

super::test_snap!(many, "hat bar ul vec ubrace obrace color ddot cancel");
