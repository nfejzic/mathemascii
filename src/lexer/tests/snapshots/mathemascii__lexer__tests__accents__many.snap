---
source: src/lexer/tests/accents.rs
expression: "Snapshot((input, tokens))"
---
hat bar ul vec ubrace obrace color ddot cancel

hat
^^^ -> Accent(Hat) at: 0 -> 3

    bar
    ^^^ -> Accent(Overline) at: 4 -> 7

        ul
        ^^ -> Accent(Underline) at: 8 -> 10

           vec
           ^^^ -> Accent(Vector) at: 11 -> 14

               ubrace
               ^^^^^^ -> Accent(Underbrace) at: 15 -> 21

                      obrace
                      ^^^^^^ -> Accent(Overbrace) at: 22 -> 28

                             color
                             ^^^^^ -> Accent(Color) at: 29 -> 34

                                   ddot
                                   ^^^^ -> Accent(DoubleDot) at: 35 -> 39

                                        cancel
                                        ^^^^^^ -> Accent(Cancel) at: 40 -> 46


